mlflow/resource_webhook_test.go

```diff
@@ -0,0 +1,229 @@
+package mlflow
+
+import (
+	"testing"
+
+	"github.com/databrickslabs/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	test_wh_job = Webhook{
+		Events:      []string{"TRANSITION_REQUEST_CREATED"},
+		Description: "Job webhook trigger",
+		Status:      "ACTIVE",
+		JobSpec: &JobSpec{
+			JobID:        "1234",
+			WorkspaceURL: "https://test.cloud.databricks.com",
+			AccessToken:  "dapi1234",
+		},
+	}
+	test_wh_id          = "12345"
+	test_wh_job_with_id = Webhook{
+		ID:          test_wh_id,
+		Events:      []string{"TRANSITION_REQUEST_CREATED"},
+		Description: "Job webhook trigger",
+		Status:      "ACTIVE",
+		JobSpec: &JobSpec{
+			JobID:        "1234",
+			WorkspaceURL: "https://test.cloud.databricks.com",
+			AccessToken:  "dapi1234",
+		},
+	}
+	test_wh_hcl = `
+	events = ["TRANSITION_REQUEST_CREATED"]
+	description = "Job webhook trigger"
+	status = "ACTIVE"
+	job_spec {
+		job_id = "1234"
+		workspace_url = "https://test.cloud.databricks.com"
+		access_token = "dapi1234"
+	}
+	`
+)
+
+func TestWebookCreate(t *testing.T) {
+	d, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:          "POST",
+				Resource:        "/api/2.0/mlflow/registry-webhooks/create",
+				ExpectedRequest: test_wh_job,
+				Response: webhookApiResponse{
+					Webhook: test_wh_job_with_id,
+				},
+			},
+			{
+				Method:   "GET",
+				Resource: "/api/2.0/mlflow/registry-webhooks/list",
+				Response: webhookListResponse{
+					Webhooks: []Webhook{test_wh_job_with_id},
+				},
+			},
+		},
+		Resource: ResourceMLFlowWebhook(),
+		Create:   true,
+		HCL:      test_wh_hcl,
+	}.Apply(t)
```

Comment:
Replace with ApplyNoError(t)

Reply:
`ApplyNoError` doesn't return data, so I can't check results. I can of course change it to return data...